src/components/live-viewers/live-viewers-manager.tsx

```diff
@@ -44,21 +55,24 @@ export class LiveViewersManager {
     }
   };
 
-  private _updateLiveViewers = () => {
+  private _updateLiveViewers = async () => {
     const entryId = this._player.config.sources.id;
-    const ks = this._player.config.session?.ks || null;
-    // make api call to get live viewers
-    this._player.provider.doRequest([{loader: GetLiveStatsLoader, params: {ks, entryId}}])
-      .then((data: Map<string, any>) => {
-        if (data && data.has(GetLiveStatsLoader.id)) {
-          const liveStatsLoader = data.get(GetLiveStatsLoader.id);
-          const liveStats = liveStatsLoader?.response;
-          // liveViewers from response is possibly false (in case of 0 viewers)
-          this._liveViewersRef?.current?.updateLiveViewers(liveStats.liveViewers || 0);
-        }
-      });
+    const requestUrl: string = this._baseRequestUrl.replace(ENTRY_ID_URL_PLACEHOLDER, entryId);
+    const liveViewers: number | undefined = await this._getLiveViewers(requestUrl);
+    typeof liveViewers === 'number' && this._liveViewersRef?.current?.updateLiveViewers(liveViewers);
```

Comment:
I think this syntax is less readable than an if. 
Inside a jsx block it's understandable to use it because you can't put an if in there, but here it's not like you have a space limitation.